templates/web/search.ftl

```diff
@@ -39,7 +39,6 @@
     <script src="${path}momentjs/moment-timezone-with-data-2012-2022.min.js?version=${UIBuildId!.now?string('Mddyyyy')}"></script>
 
     <#include "/templates/web/common/page-fragments/studio-context.ftl" />
-    <#include "/templates/web/common/page-fragments/context-nav.ftl" />
```

Comment:
I removed the static include from the template because I don't want the sidebar when in search-select mode. When in regular studio search, the sidebar is loaded here: https://github.com/eXa-online/studio-ui/blob/feature/search-childcontent/static-assets/components/cstudio-search/search.js#L89-L93
But you are right. A query string parameter might be easier. I will give it a try